science/proxmark3-iceman/Portfile

```diff
@@ -33,14 +33,35 @@ depends_lib-append  port:readline \
 
 depends_build-append \
                     port:arm-none-eabi-gcc \
-                    port:arm-none-eabi-binutils
+                    port:arm-none-eabi-binutils \
+                    port:coreutils
 
+# although the upstream repository does not have a ./configure script,
+# the buildsystem itself does rely on things usually found in ./configure
+# like for instance, pkg-config.
+#
+# this is why i (@linuxgemini) has decided to set variables related to
+# configure phase instead of making new ones before passing them
+# to build and destroot envs.
+
+set python_prefix   ${frameworks_dir}/Python.framework/Versions/3.9
+configure.python    ${prefix}/bin/python3.9
+
+configure.pkg_config \
+                    ${prefix}/bin/pkg-config
+configure.pkg_config_path-append \
+                    ${python_prefix}/lib/pkgconfig
+
```

Comment:
this does imply you should have a build dependency on "pkgconfig"?

Reply:
Linter warns with `Warning: Dependency port:pkgconfig specified multiple times in depends_build` as the `qt5` PortGroup includes it already:

https://github.com/macports/macports-ports/blob/9842c6d58d2c8fce706598308eb1cfbce0d59b45/_resources/port1.0/group/qt5-1.0.tcl#L39-L40

Reply:
right - sorry, I missed that it was already included by the qt5 PortGroup